cmd/tailscale/cli/set.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cli
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+
+	"github.com/peterbourgon/ff/v3/ffcli"
+	"tailscale.com/ipn"
+	"tailscale.com/safesocket"
+)
+
+var setCmd = &ffcli.Command{
+	Name:       "set",
+	ShortUsage: "set [flags]",
+	ShortHelp:  "Toggle preferences",
```

Comment:
I think "toggle" implies changing a boolean on/off?

"Change specified preferences"

And then in LongHelp you can say more about how it differs from "up".

Reply:
Updated, ptal.